### PR TITLE
login

### PR DESCRIPTION
**Pull request by jdimatteo**

### CODE REVIEW

**Comment by jdimatteo** [comment]
@anfedorov

File: requirements.txt [hunk 1]
```diff
@@ -20,7 +22,7 @@ flask-blueprint==1.2.2
 Flask-Login==0.4.1
 Flask-Mail==0.9.1
 Flask-Principal==0.4.0
-Flask-Security==3.0.0
+git+https://github.com/mattupstate/flask-security.git@3e15d06ee82a728d5cc53059fb28406c7bc6e7aa
```
Comment (anfedorov) [inline requirements.txt]: why not a release?

File: web_app/config/common.py [hunk 2]
```diff
@@ -1,2 +1,13 @@
-db_username = 'wegrowwealth'
-db_name = 'wegrowwealth'
+DB_USERNAME = 'wegrowwealth'
+DB_NAME = 'wegrowwealth'
+
+SECURITY_PASSWORD_HASH = 'bcrypt'
```
Comment (anfedorov) [inline web_app/config/common.py]: is scrypt not available? it's not a huge difference, but there is one and we'll get l33t points if we ever want to hired OpenBSD folks (the guy who wrote scrypt is a marginally famous OpenBSD user)

Comment (anfedorov) [reply web_app/config/common.py]: hah tpt (marginally famouse crypto junkie on hn) says

> Please stop stirring up drama about this issue. While you are technically incorrect (PBKDF2-SHA1 is faster than and thus inferior to bcrypt), it's irrelevant: all three of [scrypt, bcrypt, PBKDF2] are just fine, and you can safely pick one at random.

**Comment by anfedorov** [comment]
this is launched before it is merged? should we have a process that makes sure it's the other way around, or are PR's simply informational?

why the branch instead of a release on Flask-Security? add a comment to code explaining it so we know when to remove?

**Pull request merged**